Mathlib/Data/ENat/Lattice.lean

```diff
@@ -45,4 +46,13 @@ lemma coe_iSup : BddAbove (range f) → ↑(⨆ i, f i) = ⨆ i, (f i : ℕ∞)
 @[norm_cast] lemma coe_iInf [Nonempty ι] : ↑(⨅ i, f i) = ⨅ i, (f i : ℕ∞) :=
   WithTop.coe_iInf (OrderBot.bddBelow _)
 
+variable {s : Set ℕ∞}
+
+lemma sSup_eq_zero : sSup s = 0 ↔ s = ∅ ∨ s = {0} :=
+  sSup_eq_bot'
```

Comment:
How about having `sSup_eq_zero` mimicking `sSup_eq_bot`, and `sSup_eq_zero'` mimicking `sSup_eq_bot'`?